styles/sass/partials/_tooltip.scss

```diff
@@ -18,10 +18,10 @@
     -webkit-transition-timing-function: ease;
 
     table {
```

Comment:
do not use table directly...pls give some valid class name

Reply:
@johibkhan2  its will only apply on that that have class custom_tooltip